pkg/ndctl/region.go

```diff
@@ -93,7 +93,7 @@ func (r *Region) ActiveNamespaces() []*Namespace {
 	return r.namespaces(true)
 }
 
-//AllNamespaces returns all namespaces in the region
+//AllNamespaces returns all non-zero sized namespaces in the region
```

Comment:
Can you add a comment here explaining why it ignores namespaces with zero size? I know its in the commit message, but that's not necessarily where someone will look when reading this description.

Reply:
Done.